README.md

```diff
@@ -1 +1,166 @@
-# survey-Trials
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+  
+  <br/>
+
+  <h3><b>My Portfolio README</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [Survey-Review] <a name="about-project"></a>
+
+
+
+**This is a project we did during the Microverse Admission Trials. This project will help one do a survey and collect the neccessary information needed to accomplish a particular task. The project will be of great help and personally it made apply more knowledge very i,portant for my career growth.**
+
+## 🛠 Built With <a name="built-with"></a>
+ 
+ ### Tech Stack <a name="tech-stack"></a>
+- **HTML5**
+- **CSS3**
+### Key Features <a name="key-features"></a>
+
+
+- **This can be used to do any survey**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- HTML, CSS, and Javascript know-how.
+- Git installation
+- A text editor
+- A terminal
+- A web browser to view output e.g Google Chrome
+- An IDE e.g Visual studio code 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Commands:
+
+```sh
+  cd hello microverse
+  git clone git@github.com:Eccb7/My-Portfolio.git
+```
+
```

Comment:
I recommend you update your readme file. This would help know which repository you're referring to here while cloning. :smiley: